### DISCUSSION TITLE
Frame sequence importing with nexrender

### DISCUSSION THREAD

**Discussion by digitalpixelwizz** [inlife/nexrender#962] [Q&A]
I need to import a .png frame sequence to use as source footage in my composition. Is this possible to do with nexrender? If so what are the values in the "src" and "type" object? 

![nexrender - Copy](https://github.com/inlife/nexrender/assets/24909849/6b8a3cb9-1117-46e5-9956-1159ded5f6ec)

**Comment by sasmaster** [comment]
Hi. Have you been able to solve this one?
